src/Http/Offer/DeleteRequestHandler.php

```diff
@@ -26,7 +27,11 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         $routeParameters = new RouteParameters($request);
         $offerId = $routeParameters->getOfferId();
 
-        $this->commandBus->dispatch(new DeleteOffer($offerId));
+        try {
+            $this->commandBus->dispatch(new DeleteOffer($offerId));
+        } catch (CannotDeleteUiTPASPlace $e) {
+            throw $e->toApiProblem();
```

Comment:
There is no need to call this, I think the default ErrorLogger will handle this

Reply:
Ah, oké, did not know this. Changed it, and tested it again - it works.